### PR TITLE
replace docker image place holder using sed

### LINKED ISSUE #61: Docker image creation failed
due to incorrect docker image at the point in time when `docker build` is called: https://github.com/zalando/ghe-backup/blob/master/Jenkinsfile#L106

caused most likely by https://github.com/zalando/ghe-backup/blob/master/Jenkinsfile#L103

### PR DESCRIPTION
**Pull request by lotharschulz**
fix #61

### CODE REVIEW

**Comment by lotharschulz** [comment]
build succeeded and build pushed to pierone instance

**Comment by lotharschulz** [comment]
pls review: @jmcs @rashamalek

File: Jenkinsfile [hunk 1]
```diff
@@ -97,10 +97,10 @@ if (env.BRANCH_NAME == 'master') {
 
 //  Builds the docker image and returns the full name of the new image:
 def docker(String dockerRepo, String fullImageName, String dockerfile, boolean pushImage) {
-    sh "git status"
     if (pushImage == true) {
         sh "/tools/run :stups -- scm-source"
-        sh "/tools/run :stups -- echo 'COPY scm-source.json /' >> $dockerfile"
+        sh "/tools/run :stups -- sed -i 's&.*#PLACEHOLDER_4_COPY_SCM_SOURCE_JSON.*&COPY scm-source.json /scm-source.json&' DockerfileBus"
```
Comment (rashamalek) [inline Jenkinsfile:102]: Could use this as well, may be more readable 
```
sed -i 's/#PLACEHOLDER_4_COPY_SCM_SOURCE_JSON/COPY scm-source.json \/scm-source.json/'
```

**Comment by rashamalek** [comment]
👍

**Comment by lotharschulz** [comment]
:+1:

File: Jenkinsfile (hunk 1 above)
Comment (lotharschulz) [reply Jenkinsfile:102]: I tried that with our jenkins and it failed.

**Pull request merged**